quantum/debounce/tests/sym_defer_g_tests.cpp

```diff
@@ -236,3 +236,22 @@ TEST_F(DebounceTest, OneKeyDelayedScan4) {
     time_jumps_ = true;
     runEvents();
 }
+
+TEST_F(DebounceTest, AsyncTickOneKeyShort1) {
+    addEvents({
+        /* Time, Inputs, Outputs */
+        {0, {{0, 1, DOWN}}, {}},
+
+        {5, {}, {{0, 1, DOWN}}},
+        /* 0ms delay (fast scan rate) */
+        {5, {{0, 1, UP}}, {}},
+
+        {10, {}, {{0, 1, UP}}},
+    });
+    /*
+     * Debounce implementations should never read the timer more than once per invocation,
+     * due to some hardware platforms having an asynchronous timer (e.g. ARM).
+     */
```

Comment:
Actually all hardware platforms have asynchronous timers (`avr` and `arm_atsam` update a millisecond counter from a timer interrupt, `chibios` relies on the ChibiOS system timer, which also gets updated in an asynchronous way).  The only timer implementation that was guaranteeing that the timer value does not change without some explicit action done by the test code was the `test` platform before your changes in this PR.

Reply:
Yes, I thought that was the case, but I didn't realize it until I dug a little deeper into the code for those platforms. I'll change the comment.